data/api/identity/v2_pubkey.yaml

```diff
@@ -43,7 +43,7 @@ paths:
                 properties:
                   public_key:
                     type: string
-                    description: Unpadded Base64 encoded public key.
+                    description: Public key using standard or URL-safe base64 encoding.
```

Comment:
Given that Sydent doesn't currently return a url-safe key here (see https://vector.im/_matrix/identity/v2/pubkey/ed25519:0), maybe we could avoid changing this.

Reply:
I reverted the change, but added a link to the definition of unpadded base64.